util/common.js

```diff
@@ -0,0 +1,21 @@
+const { flatten } = require("flat");
+
+class CommonUtils {
+  static objectDiff(obj1, obj2) {
+    const flatObj1 = flatten(obj1 || {});
+    const flatObj2 = flatten(obj2 || {});
+    const allKeys = new Set(
+      Object.keys(flatObj1).contact(Object.keys(flatObj2))
```

Comment:
`contact` or `concat` ?

Reply:
Yes, typo

Reply:
fixed

Reply:
I have setup local files to test this and comparison it working now.